Test/Invoke-WhiskeyExec.Tests.ps1

```diff
@@ -48,7 +48,15 @@ function GivenADirectory
     New-Item -Path (Join-Path -Path (Get-BuildRoot) -ChildPath $DirectoryPath) -ItemType 'Directory' -Force | Out-Null
 }
 
-function GivenExecutableFile
+function GivenExecutable

```

Comment:
This function doesn't do anything and isn't used in any tests.

Reply:
‽